### ISSUE TITLE
Accessibility problems with Format Code Block dialog

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7081]
### System details

    RStudio Edition : Server
    RStudio Version : 1.4.453
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- enable RMD visual editor and open an RMD file
- click the Format/Code Block button
- attempt to use the dialog via keyboard and/or screen reader

### Describe the problem in detail
- in the initial view ("Language" tab selected) the tab control cannot be reached via the Tab key; forward tab order goes from the Cancel button back to the Language field, bypassing the tab control. If you Shift+Tab you can reach the Tab control. Suggests dialog is constructed in a way that confuses the (attempt) at determining first/last focusable control and mixes up the wrap-around behavior.
- the Language field is unlabelled (for a screen reader user); the visible label is not programmatically associated with the field

<img width="403" alt="screenshot of format code block dialog with Language tab selected" src="https://user-images.githubusercontent.com/10569626/84209746-c6b20700-aa6b-11ea-94e5-2f0bc0b4f25a.png">

- select the "Attributes" tab
- tabbing forward through the dialog gets stuck on the Cancel button, no wrap-around
- tabbing backwards from the dialog gets stuck on the Tab control
- the first three controls are unlabeled and are not usable by a screen reader user; you can also verify this by clicking on the label; a properly associated label will cause the control to get focus when you click on it
- the fourth control (Other) is labeled with the text "Attributes" and not the label that is shown for it
- the example text for each field is also not usable via screen reader unless user switches screen reader into virtual cursor mode and navigates that way (not how a form-like dialog is normally navigated)

<img width="402" alt="screenshot of format code block dialog with Attributes tab selected" src="https://user-images.githubusercontent.com/10569626/84209858-0bd63900-aa6c-11ea-9b2c-57803d649977.png">

### Describe the behavior you expected
- although the visual editor itself is not currently accessible to screen reader users, we should ensure the rest of the components meet minimum accessibility standards if only to prevent future reuse of these components from propagating bad practices and so the feature lights up if and when we fix the editor

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
